File: py/IEC61970/Base/DC/DCEquipmentContainer.py
# Converted by an OPENAI API call using model: gpt-3.5-turbo-1106 on Fri Dec 15 17:22:57 2023
from CIM_STD_PYTHON.TC57CIM.IEC61970.Base.Core.EquipmentContainer import EquipmentContainer
from CIM_STD_PYTHON.TC57CIM.IEC61970.Base.DC.DCTopologicalNode import DCTopologicalNode

class DCEquipmentContainer(EquipmentContainer):
    """
    A modeling construct to provide a root class for containment of DC as well as
    AC equipment. The class differs from the EquipmentContaner for AC in that
    it may also contain DCNodes. Hence it can contain both AC and DC equipment.
    """
    def __init__(self) -> None:
        super().__init__()

File: java/IEC61970/Base/DC/DCEquipmentContainer.java
package IEC61970.Base.DC;

import IEC61970.Base.Core.EquipmentContainer;

/**
 * A modeling construct to provide a root class for containment of DC as well as
 * AC equipment. The class differ from the EquipmentContaner for AC in that it may
 * also contain DCNodes. Hence it can contain both AC and DC equipment.
 * @author selaost1
 * @version 1.0
 * @created 25-Dec-2023 8:31:55 PM
 */
public class DCEquipmentContainer extends EquipmentContainer {

	/**
	 * The topological nodes which belong to this connectivity node container.
	 */
	public DCTopologicalNode DCTopologicalNode;

	public DCEquipmentContainer(){

	}
}//end DCEquipmentContainer